includes/class-admin.php

```diff
@@ -46,6 +49,30 @@ public static function admin_menu() {
 		}
 	}
 
+	/**
+	 * Display admin menu notices about configuration problems or conflicts
+	 */
+	public static function admin_notices() {
+		$permalink_structure = \get_option( 'permalink_structure' );
+		if ( empty( $permalink_structure ) ) {
+			$admin_notice = \__( 'You are using the ActivityPub plugin without setting a permalink structure.  This will prevent ActivityPub from working.  Please set a permalink structure.', 'activitypub' );
+			self::show_admin_notice( $admin_notice, 'error' );
+		}
+	}
+
+	/**
+	 * Display one admin menu notice about configuration problems or conflicts
+	 */
+	private static function show_admin_notice( $admin_notice, $level ) {
+		?>
+
+		<div class="notice notice-<?php echo wp_kses( $level, 'data' ); ?>">
```

Comment:
I would use `esc_attr` instead of `wp_kses` here, because we do not want to allow html at all!